GenerateKeywords.cmake

```diff
@@ -82,4 +82,5 @@ add_custom_command( OUTPUT
 add_custom_command(OUTPUT
   ${_target_output}
   DEPENDS ${_tmp_output}
+  DEPENDS copy_python
```

Comment:
What does this do if a build configuration does not enable Python bindings?  Does the `copy_python` target/custom command exist unconditionally or only if the user sets `OPM_ENABLE_PYTHON=YES`?

Reply:
good catch. it works with make but not with ninja. i'll have to rethink.

Reply:
this is tested with ninja and cmake, with or without python and works in all cases.

Reply:
> this is tested with ninja and cmake, with or without python and works in all cases.

Excellent, thanks a lot.  I'll defer final approval and merging to @blattms.